vmdb/app/controllers/ems_infra_controller.rb

```diff
@@ -52,7 +52,18 @@ def scaling
           scale_parameters_formatted << {"name" => p.name, "value" => scale_parameters[p.name]}
         end
       end
-      if scale_parameters_formatted.length > 0
+
+      begin
+        # Check if stack is ready to be updated
+        update_ready = @stack.update_ready?(@stack.raw_status.first)
```

Comment:
This makes me think that either the `update_ready?` method should be a static method (because it receives the state as a parameter), or that the `update_ready?` method should not take in a parameter, and should encapsulate the logic for looking up the state itself.  I lean toward the second approach (leave `update_ready?` an instance method and remove the parameter).